src/SSW.Rewards.Application/Common/Behaviours/PerformanceBehaviour.cs

```diff
@@ -22,7 +22,7 @@ public class PerformanceBehaviour<TRequest, TResponse> : IPipelineBehavior<TRequ
         _userService = userService;
     }
 
-    public async Task<TResponse> Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate<TResponse> next)
+    public async Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TResponse> next, CancellationToken cancellationToken)
```

Comment:
pass the cancellation token to any of the other async/await methods like `_userService.GetUserId()`

Reply:
That definitely needs doing, I've added an issue to check that across the whole codebase here: https://github.com/SSWConsulting/SSW.Rewards.API/issues/65

Reply:
Done.